extensions/idp/pkg/command/server.go

```diff
@@ -29,6 +40,15 @@ func Server(cfg *config.Config) *cli.Command {
 				fmt.Printf("%v", err)
 				os.Exit(1)
 			}
+
+			if cfg.IDP.EncryptionSecretFile != "" {
+				if err := ensureEncryptionSecretExists(cfg.IDP.EncryptionSecretFile); err != nil {
+					return err
+				}
+				if err := ensureSigningPrivateKeyExists(cfg.IDP.SigningPrivateKeyFiles); err != nil {
+					return err
+				}
+			}
```

Comment:
the idm creates it's certificates during the `Action` (https://github.com/owncloud/ocis/blob/889acbfbf05787a9ed20c4d0a37b4ccfc2710401/extensions/idm/pkg/command/server.go#L67-L71). Should we do the same for the IDP?

Reply:
Maybe the idm should also do this in the before function? At least IMO it would make sense since `Before` is doing preparations for the `Action`.